src/styles/blocks/button.scss

```diff
@@ -0,0 +1,39 @@
+.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 56px;
+
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 14px;
+  text-decoration: none;
+  border-radius: 8px;
+  border: 2px solid $color-blue;
+
+  cursor: pointer;
+  transition: all 0.3s;
```

Comment:
button has box-shadow in default mode too
see design kit
https://www.figma.com/file/nHz8bflIwJaWP3P99vKTH5/miami_home_new?node-id=16039%3A46